yo/main.py

```diff
@@ -1526,6 +1526,51 @@ def run_for_instance(self, inst: YoInstance) -> None:
             )
 
 
+class CopyIdCmd(SingleInstanceCommand):
+    name = "copy-id"
+    description = "copy an SSH public key onto an instance using ssh-copy-id"
+
+    def add_args(self, parser: argparse.ArgumentParser) -> None:
+        super().add_args(parser)
+        parser.add_argument(
+            "-i",
+            "--identity-file",
+            type=str,
+            required=True,
```

Comment:
`ssh-copy-id` can be run without the `-i` argument, in which case it copies the default SSH key.  I think that would be useful to support here.
```suggestion
            required=False,
```